frontend/src/components/form/base/ETimeField.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <EParseField
+    ref="input"
+    :value="value"
+    :format="format"
+    :parse="parse"
+    :serialize="serialize"
+    :deserialize="deserialize"
+    :required="required"
+    :vee-id="veeId"
+    :vee-rules="veeRules"
+    reset-on-blur
+    v-bind="$attrs"
+    v-on="$listeners"
+    @input="$emit('input', $event)"
+  >
+    <!-- passing through all slots -->
+    <slot v-for="(_, name) in $slots" :slot="name" :name="name" />
+    <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
+      <slot v-if="name !== 'prepend'" :name="name" v-bind="slotData" />
+    </template>
+  </EParseField>
+</template>
+
+<script>
+import { formComponentMixin } from '@/mixins/formComponentMixin.js'
+import parseTime from '@/common/helpers/dayjs/parseTime.js'
+
+export default {
```

Comment:
I assume the goal is that the ETimeField has the same relation to the ETimePicker as the EColorField to the EColorPicker. But its not used right now...?

Reply:
Could be, but I more have it in mind to just use it more flexible. e.g without the timepicker

Reply:
Will be used for later restyling of the Schedule Entry edit experience.